src/sirinternal.c

```diff
@@ -833,10 +850,10 @@ const char* _sir_format(bool styling, sir_options opts, sirbuf* buf) {
         if (styling)
             (void)_sir_strncat(buf->output, SIR_MAXOUTPUT, SIR_ESC_RST, SIR_MAXSTYLE);
 
-#if defined(SIR_EOL_CRLF)
-        (void)_sir_strncat(buf->output, SIR_MAXOUTPUT, "\r\n", 2);
```

Comment:
This can be consolidated to use `SIR_EOL` now

Reply:
Done.